docker-compose.yml

```diff
@@ -1,8 +1,37 @@
-# version: '3.9'
+version: "3.9"
+services:
+  revent-db:
+    container_name: revent-db
+    image: postgres:latest
+    # restart: always
+    env_file:
+      - ".env"
+    ports:
+      - "5500:5432"
+    volumes:
+      - ./data/db:/var/lib/postgresql/data
 
-# services:
-#   web:
-#     build: .
-#     command: python manage.py runserver 0.0.0.0:8000
-#     ports:
-#       - 8000:8000
+  revent-be:
```

Comment:
what is the difference between this service and web? do we need this?

Reply:
removed web instead, is it ok?